packages/core-bridge/index.d.ts

```diff
@@ -161,16 +161,14 @@ export interface WorkerOptions {
 
   maxConcurrentActivityTaskExecutions: number;
   maxConcurrentWorkflowTaskExecutions: number;
-  maxConcurrentWorkflowTaskPolls: number;
-  maxConcurrentActivityTaskPolls: number;
+  maxConcurrentLocalActivityExecutions: number;
+
   /**
-   * `maxConcurrentWorkflowTaskPolls` * this number = the number of max pollers that will
-   * be allowed for the nonsticky queue when sticky tasks are enabled. If both defaults are used,
-   * the sticky queue will allow 4 max pollers while the nonsticky queue will allow one. The
-   * minimum for either poller is 1, so if `max_concurrent_wft_polls` is 1 and sticky queues are
-   * enabled, there will be 2 concurrent polls.
+   * If set to `false` this worker will only handle workflow tasks and local activities, it will not
+   * poll for activity tasks.
```

Comment:
```suggestion
   * poll for activity tasks.
   * 
   * @default true
```

Reply:
There's no default in this interface, it's in `WorkerOptionsWithDefaults`